src/main/java/com/fasterxml/jackson/databind/type/TypeFactory.java

```diff
@@ -1,28 +1,41 @@
 package com.fasterxml.jackson.databind.type;
 
+import com.fasterxml.jackson.core.type.TypeReference;
```

Comment:
Would it be possible to avoid changing import statement ordering? Jackson style declares imports starting with platform (JDK) ones. Similarly, auto-expansion should not be use for more than 5 imports?

Reply:
is there a style document or file somewhere? i'd be happy to fix my intellij settings since this isn't the first time it has come up

Reply:
Good question: not yet. I'll create an issue to create a Wiki page under `jackson` project. There aren't tons of things to list but it'd be necessary to outline these.

Reply:
Created https://github.com/FasterXML/jackson-databind/issues/4156 as follow-up

Reply:
Jackson Coding Style Guide is here, ftw: https://github.com/FasterXML/jackson/blob/master/contribution/jackson-coding-style-guide.md